### ISSUE TITLE
Error when building Arcade 2.6.14

### ISSUE THREAD

**Issue by jgawarecki** [pythonarcade/arcade#1225]
## Bug Report
unable to build arcade 2.6.14 
I was able to fix the issue by running
'pip install -Iv arcade==2.6.13'

Perhaps this bug only appears when installing arcade 2.6.14 with no prior versions installed
## System Info
Running ubuntu linux with the pycharm IDE in a virtual environment

Run this and paste the output here: python -m arcade
```
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 185, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/lib/python3.8/runpy.py", line 144, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/usr/lib/python3.8/runpy.py", line 111, in _get_module_details
    __import__(pkg_name)
  File "/home/joli/Documents/Python/venv/lib/python3.8/site-packages/arcade/__init__.py", line 250, in <module>
    from .joysticks import get_game_controllers
  File "/home/joli/Documents/Python/venv/lib/python3.8/site-packages/arcade/joysticks.py", line 1, in <module>
    import pyglet.input
  File "/home/joli/Documents/Python/venv/lib/python3.8/site-packages/pyglet/input/__init__.py", line 179, in <module>
    from .evdev import get_devices as evdev_get_devices
  File "/home/joli/Documents/Python/venv/lib/python3.8/site-packages/pyglet/input/evdev.py", line 509, in <module>
    class EvdevControllerManager(ControllerManager, XlibSelectDevice):
  File "/home/joli/Documents/Python/venv/lib/python3.8/site-packages/pyglet/input/evdev.py", line 583, in EvdevControllerManager
    def get_controllers(self) -> list[Controller]:
TypeError: 'type' object is not subscriptable
```
### Actual behavior:
unable to run 'import arcade'
the above message is shown

### Expected behavior:
no error

### Steps to reproduce/example code:
create a new environment with no packages installed.  
run in terminal 'pip install arcade'
run in a .py file "import arcade"

**Comment by pvcraven** [comment]
Can you install pyglet 2.0.dev15 and run the code in this github issue and see if it fails?

https://github.com/pyglet/pyglet/issues/607

**Comment by pushfoo** [comment]
**tl;dr I think this is a different pyglet bug, but try the pyglet version suggested first**

The python version in the issue is shown as 3.8. I think subscripting the base `list` type was [added in a 3.9 release](https://docs.python.org/3/whatsnew/changelog.html#python-3-9-0-alpha-6):
> [bpo-39481](https://bugs.python.org/issue?@action=redirect&bpo=39481): Implement PEP 585. This supports list[int], tuple[str, …] etc.

If the new pyglet version doesn't fix the problem, it should be reported as an upstream pyglet bug given they claim support for 3.6+.

**Comment by jgawarecki** [comment]
The issue persists with pyglet 2.0.dev15 but the issue was fixed in my tests using pyglet 2.0.dev14 and pyglet 2.0.dev13

**Comment by pushfoo** [comment]
This definitely looks like an upstream pyglet issue. I'll try to file an issue and PR on their project today.

**Comment by pushfoo** [comment]
@jgawarecki **tl;dr if you're impatient, everything should work if you upgrade to the latest Python 3.10 release. Don't use 3.11 because dependencies have issues with it.**

I made a PR to address this issue in Pyglet, but I'm not sure how long it'll take to approve and come out in a new dev release. The 2.0.15dev release of Pyglet fixed some issues we had, but I'm not sure which ones they were or if they'll affect you directly while using older versions.

Upgrading Python to the latest 3.10 release might be more reliable than messing with pyglet versions. 3.9 might work as well. Pillow (and possibly other dependencies) have some issues on 3.11 if I remember correctly.

**Comment by einarf** [comment]
https://github.com/pyglet/pyglet/issues/614

**Comment by einarf** [comment]
We should have detected this in our ci / github actions. That's something we need to look into improving.

**Comment by pushfoo** [comment]
@jgawarecki important correction: the latest 3.9 release may work as well if you're avoiding 3.10 and up for some reason. I edited the earlier posts to link to the correct version in the changelog.

The PR to fix pyglet's side of this just got merged into their `master` branch, but I'm not sure when the next dev release will be.

**Comment by einarf** [comment]
2.6.15 is released fixing this issue
